### PR TITLE
Feature/new stories table

### PR DESCRIPTION
**Pull request by nico-herrera**
# Description

Fixes #

- What work was done?
Created a StoriesNew table and migrated data from Stories "Title" row into StoriesNew.Title
- Why was this work done?
 It was a necessary step in order to restructure the database schema.
- What feature / user story is it for?
Feature: Redesign Story Squad Schema || User Story: NA Backend Feature
- Any relevant links or images / screenshots
N/A

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

## Change Status

- [ x] Complete, but not tested (may need new tests)

## Has This Been Tested

- [ ] Yes
- [x ] No
- [ ] Not necessary

## Checklist

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] My code has been reviewed by at least one peer
- [ x] My changes generate no new warnings
- [x ] There are no merge conflicts

### CODE REVIEW

**Review by lukecparr — CHANGES_REQUESTED** [review]
Looks like you'll need to run the linter and fix these issues. I know there are some instructions in the 'CONTRIBUTING.md' file if needed.

File: data/migrations/20210802205311_stories_new.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('StoriesNew', (tbl) => {
```
Comment (ChadDiaz) [inline data/migrations/20210802205311_stories_new.js]: Look at the table name. I feel the convention overall has been to use a hyphen.

**Review by ChadDiaz — APPROVED** [review]
Code looks clean.  Great job!

**Review by lukecparr — APPROVED** [review]

**Comment by lukecparr** [comment]
Check pass and everything looks good 👍🏻

**Review by nico-herrera — COMMENTED** [review]
i have formatted my code

**Pull request merged**